virttest/bootstrap.py

```diff
@@ -350,6 +351,15 @@ def _forced_or_detected(forced, detected):
                                       "u%s" % detected.release)
         arch = _forced_or_detected(options.vt_host_distro_arch,
                                    "Host_arch_%s" % detected.arch)
+        vendor = cpu.get_cpu_vendor_name()
+        revision = ""
+        if 'powerpc' in cpu.get_cpu_arch():
```

Comment:
You should only call the `get_cpu_arch` once and store it in a variable...

Reply:
Also please have a look at: https://github.com/avocado-framework/avocado/pull/3350#pullrequestreview-331926737 as I don't think we should need any branches here. The Avocado cpu utils should provide a common method to get sane representation of any kind of CPU.

Reply:
sure...